packages/next-urql/src/with-urql-client.tsx

```diff
@@ -67,7 +67,7 @@ export function withUrqlClient(
       }
 
       // Run the wrapped component's getInitialProps function.
-      let pageProps;
+      let pageProps = {} as any;
```

Comment:
Here's the bugger that broke everything 😓

Reply:
nice catch. i think this used to be typed as the generic `IP` over `withUrqlClient`, could we use that?

Reply:
Yeah I relaxed the types a bit in 0.3.0 since we now properly support wrapping `_app.js`, which made dealing with that generic a _nightmare_. If anyone _really_ wants to fix those types I'd be happy to review a PR, but creating a nice abstraction around both `_app.js` and traditional page components is quite tough. Just the reality of Next's funky API.